manifests/t/tangshimin/MuJing/2.2.16/tangshimin.MuJing.locale.en-US.yaml

```
@@ -0,0 +1,41 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: tangshimin.MuJing
+PackageVersion: 2.2.16
+PackageLocale: en-US
+Publisher: 深圳市龙华区幕境网络工作室
+PublisherUrl: https://github.com/tangshimin
+PublisherSupportUrl: https://github.com/tangshimin/MuJing/issues
+PrivacyUrl: https://github.com/tangshimin/MuJing/blob/master/Privacy%20Policy.md
+PackageName: 幕境
+PackageUrl: https://github.com/tangshimin/MuJing
+License: GPL-3.0
+LicenseUrl: https://github.com/tangshimin/MuJing/blob/master/LICENSE
+Copyright: Copyright 2024 Shimin Tang. All rights reserved.
+ShortDescription: Generate thesauruses (word book) from MKV videos so that each word has a specific context, and then memorize the words, sentences, and subtitles through the context.
+Description: |-
+  Learn English by immersion.
+  Use any movies, series or documents you like to generate your thesaurus.
+  When memorising words, you can better understand and remember words by practising spelling and watching relevant video clips.
+  When playing movies, you can review the words in the thesaurus through Dammu, making word memorisation less of a chore.
+Tags:
+- caption
+- education
+- english
+- learn
+- study
+- subtitle
+- thesaurus
+- vocabulary
+- word
+ReleaseNotes: |-
+  - 在记忆单词界面，单词拼写错误后，不用重新输入了
+  - 增加快捷键 ←、→切换单词
+  - 保存记忆单词界面的视频播放器的大小和位置

```

```suggestion
  - In the word memory interface, if a word is spelled incorrectly, there is no need to re-enter it.
  - Added shortcut keys ←, → to switch words.
  - Save the size and position of the video player in the memory word interface.
```
Release notes should match localization. 
(Internet Translation - feel free to update.)